# Homo Sapien Malate Dehydrogenase 2

# Uniprot ID: P40926

# Varation: Mimic variant and PTM at site 155

## Description:

Asp 155 is the location of the mimic variant in MDH2. TPO 155 is the location of the PTM which was phosphorylated in MDH2. No previous research has been done on modifications in MDH2 at this location. 

# Comparison of MDH2 model to mimic varaint and PTM
1. Alignment of unmodified MDH2 (brown and grey), MDH2 with mimic variant (magenta and violet) , and MDH2 with PTM (green and yellow) 
![image](https://github.com/user-attachments/assets/9654fe68-3f38-4cf4-8533-9d109c6419c3)

2. Modification site alignment with MDH2
![image](https://github.com/user-attachments/assets/9c461303-f79f-4ced-9047-9f658f538df3)
With the addition of the PTM, TPO 155 interacts with Ser 45, Lys 21, Thr 49, Ile 47,Glu 159, Val 151 through hydrogen bonds and PHE 156 through a hydrophobic interaction. The mimic variant has the same weak interactions as the unmodified MDH2 except for the deletion of Hydrogen bonding between His 46.

# Effect of the sequenct variant and PTM on MD dynamics
 The RMSD in the final frame from MD simulations of the mimic variant was 1.1 Ã… The unmodified MDH2 is shown in magenta while the mimic variant is shown in red. 
 ![image](https://github.com/user-attachments/assets/25e4242e-0ea8-4796-8dcc-bd473f0432ae)


 The overall structure of the protein is similar. The modification site in the mimic variant is Asp 155 compared to Thr 155. The interactions are similar except for the unmodifed version having an additional hydrogen bond at His 46.
 ![image](https://github.com/user-attachments/assets/66066dd2-0af5-4605-8f2a-df4a0d063e14)


 ## Comparison of the enzyme dynamics
 After simulation the dynamics from the root mean square flucation (RMSF) were compared. There are differences between the mimic varaint (blue) and the unmodified (orange) around amino acids 90, 210,and 628. These are loop site that are in proximity to the active site His 176. Differnces in these loop sites correspond to the changes made my the modification to the active site itself. 
![image](https://github.com/user-attachments/assets/9945bc04-f02a-49b7-b45c-c22363262f1f)

The loop sites are shown in the ball and stick conformations. The loop site within the yellow circle is the active site loop. The yellow eclipse shows the proximitiy between the binding site and active site. 
![image](https://github.com/user-attachments/assets/53c01d7a-ae2e-4f5d-934d-c721836b1585)

 The modification signficantly affected the pKa of the active site at Histidine 176. The unmodified MDH2 shown in blue flucuates around a pKa of 4. The mimic variant flucuates around a pKa of 13. The change in pKa at the active site could signficantly affect the metabolic pathway used during the Krebs cycle by changing the efficenecy of the catalytic reaction to covert malate to oxolaecetate. 
![image](https://github.com/user-attachments/assets/2bc31400-c373-4296-a53a-0617b4f3ed88)

## Comparison of MD simulated mimic variant and authentic PTM
The RMSD of the simulated mimic varaint and PTM was 1.45. Overall the strucutre changes are minimal in structure adn position. 
![image](https://github.com/user-attachments/assets/dc2f9e99-185e-4cac-bc18-0710f8aeca93)

## Colab notebook links
Copy_of_MD_simulation_Step1
Copy_of_mdanalysis_colab_Step2
## Authors

Elizabeth Casey

## Deposition Date
12/6/2024

## License

Shield: [![CC BY-NC 4.0][cc-by-nc-shield]][cc-by-nc]

This work is licensed under a
[Creative Commons Attribution-NonCommercial 4.0 International License][cc-by-nc].

[![CC BY-NC 4.0][cc-by-nc-image]][cc-by-nc]

[cc-by-nc]: https://creativecommons.org/licenses/by-nc/4.0/
[cc-by-nc-image]: https://licensebuttons.net/l/by-nc/4.0/88x31.png
[cc-by-nc-shield]: https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg

## References
 UniProt. www.uniprot.org. https://www.uniprot.org/uniprotkb/P40926/entry. 
